config/chassis_modules.py

```diff
@@ -110,8 +110,9 @@ def shutdown_chassis_module(db, chassis_module_name):
 
     if not chassis_module_name.startswith("SUPERVISOR") and \
        not chassis_module_name.startswith("LINE-CARD") and \
-       not chassis_module_name.startswith("FABRIC-CARD"):
-        ctx.fail("'module_name' has to begin with 'SUPERVISOR', 'LINE-CARD' or 'FABRIC-CARD'")
+       not chassis_module_name.startswith("FABRIC-CARD") and \
```

Comment:
We need to perform additional validation to check if the chassis_module_name is actually present or not, if user executes `config chassis modules startup DPU5` on a system which does not have DPU5, this will cause crash in chassisd for the `SmartSwitchConfigManagerTask` in chassisd preventing further startup or shutdown calls (even though output of the command would be `Starting up chassis module DPU1` or `Shutting down chassis module DPU1` even though the only operation which is done is addition/removal from the CONFIG_DB )

Reply:
@gpunathilell will add this check